spec/commitments_deprecater_spec.rb

```diff
@@ -0,0 +1,197 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require_relative "../bin/commitments_deprecater"
+
+class Mockmitment
+  attr_reader :local_id
+
+  def initialize(id)
+    @local_id = id
+  end
+end
+
+RSpec.describe CommitmentsDeprecater do
+  let(:org_1) { "umich" }
+  let(:org_2) { "smu" }
+  let(:ocn_1) { 111 }
+  let(:ocn_2) { 222 }
+  let(:local_id_1) { "i111" }
+  let(:local_id_2) { "i222" }
+  let(:status_1) { "E" }
+  let(:status_2) { "M" }
+  let(:line) { [org_1, ocn_1, local_id_1, status_1].join("\t") }
+  let(:dep) { described_class.new }
+
+  let(:empty_cluster) { create(:cluster) }
+
+  def make_commitment(ocn, org, local_id)
+    # Make and cluster a commitment
+    com = build(:commitment, ocn: ocn, organization: org, local_id: local_id)
+    cluster = Clustering::ClusterCommitment.new(com).cluster.tap(&:save)
+    cluster.commitments.first
+  end
+
+  def make_holding(ocn, org, local_id)
```

Comment:
This returns a cluster, not a holding, but also it doesn't seem to be used, so perhaps we can remove it?

Reply:
yep, that was a copy-paste leftover, is gone now